seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -596,7 +596,7 @@
         "assert($(\"p\").length > 1, 'message: You need 2 <code>p</code> elements with Kitty Ipsum text.');",
         "assert(code.match(/<\\/p>/g) && code.match(/<\\/p>/g).length === code.match(/<p/g).length, 'message: Make sure each of your <code>p</code> elements has a closing tag.');",
         "assert.isTrue((/Purr\\s+jump\\s+eat/gi).test($(\"p\").text()), 'message: Your <code>p</code> element should contain the first few words of the provided additional <code>kitty ipsum text</code>.');",
-        "assert($(\"p:not([class])\").length === 1, 'message: Do not add a class attribute to the second <code>p</code> element, without removing it from the first one.');",
+        "assert($(\"p:not([class])\").length === 1, 'message: Do not add a class attribute to the second <code>p</code> element. The first <code>p</code> should have a class attribute and it's content should remain red.);",
```

Comment:
`red.);` has a missing `'`.
Should be `red.');`

Your code gives :

![image](https://cloud.githubusercontent.com/assets/1884376/13812751/ebc2c4e8-eba3-11e5-87b6-0eb4217b0053.png)